File: client/iih_stub/iih.ci_stub/iih.ci.rcm/contagionmessage/d/ContagionMessageDTO.cs
using System;
using xap.mw.core.data;
using xap.mw.coreitf.d;

namespace iih.ci.rcm.contagionmessage.d
{
    /// <summary>
    /// ContagionMessageDTO 的摘要说明。
    /// </summary>
    public class ContagionMessageDTO : BaseDTO {

        public ContagionMessageDTO() {
 
        }

        /// <summary>
        /// 传染病主键
        /// </summary>
		public string Id_contagion {
            get { return getAttrVal<string>("Id_contagion",null); }
            set { setAttrVal<string>("Id_contagion", value); }
        }

        /// <summary>
        /// 保存时间
        /// </summary>
		public string Save_time {
            get { return getAttrVal<string>("Save_time",null); }
            set { setAttrVal<string>("Save_time", value); }
        }

        /// <summary>
        /// 填报科室
        /// </summary>
		public string Report_dep {
            get { return getAttrVal<string>("Report_dep",null); }
            set { setAttrVal<string>("Report_dep", value); }
        }

        /// <summary>
        /// 填报医生
        /// </summary>
		public string Report_doc {
            get { return getAttrVal<string>("Report_doc",null); }
            set { setAttrVal<string>("Report_doc", value); }
        }

        /// <summary>
        /// 患者姓名
        /// </summary>
		public string Name_pat {
            get { return getAttrVal<string>("Name_pat",null); }
            set { setAttrVal<string>("Name_pat", value); }
        }

        /// <summary>
        /// 疾病名称
        /// </summary>
		public string Name_disease {
            get { return getAttrVal<string>("Name_disease",null); }
            set { setAttrVal<string>("Name_disease", value); }
        }

        /// <summary>
        /// 自定义1
        /// </summary>
		public string Def1 {
            get { return getAttrVal<string>("Def1",null); }
            set { setAttrVal<string>("Def1", value); }
        }

        /// <summary>
        /// 自定义2
        /// </summary>
		public string Def2 {
            get { return getAttrVal<string>("Def2",null); }
            set { setAttrVal<string>("Def2", value); }
        }

        /// <summary>
        /// 自定义3
        /// </summary>
		public string Def3 {
            get { return getAttrVal<string>("Def3",null); }
            set { setAttrVal<string>("Def3", value); }
        }
        /// <summary>
        /// 返回属性列表
        /// </summary>
        /// <returns></returns>
        public override string[] getAttrNames()
        {
            return new string[]{ "Id_contagion", "Save_time", "Report_dep", "Report_doc", "Name_pat", "Name_disease", "Def1", "Def2", "Def3"};
        }
        
        /// <summary>
        /// 返回全路径DO类名
        /// </summary>
        /// <returns></returns>
        public override string getFullClassName()
        {
            return "iih.ci.rcm.contagionmessage.d.ContagionMessageDTO";
        }
    }
}

File: server/iih_pub/iih.ci_pub/src/main/java/iih/ci/rcm/contagionmessage/d/ContagionMessageDTO.java
package iih.ci.rcm.contagionmessage.d;

import xap.mw.core.data.*;
import xap.mw.coreitf.d.*;
import java.math.BigDecimal;

/**
 * 传染病消息DTO DTO数据 
 * 
 */
public class ContagionMessageDTO extends BaseDTO {
	private static final long serialVersionUID = 1L;
	
	/**
	 * 传染病主键
	 * @return String
	 */
	public String getId_contagion() {
		return ((String) getAttrVal("Id_contagion"));
	}
	/**
	 * 传染病主键
	 * @param Id_contagion
	 */
	public void setId_contagion(String Id_contagion) {
		setAttrVal("Id_contagion", Id_contagion);
	}
	/**
	 * 保存时间
	 * @return String
	 */
	public String getSave_time() {
		return ((String) getAttrVal("Save_time"));
	}
	/**
	 * 保存时间
	 * @param Save_time
	 */
	public void setSave_time(String Save_time) {
		setAttrVal("Save_time", Save_time);
	}
	/**
	 * 填报科室
	 * @return String
	 */
	public String getReport_dep() {
		return ((String) getAttrVal("Report_dep"));
	}
	/**
	 * 填报科室
	 * @param Report_dep
	 */
	public void setReport_dep(String Report_dep) {
		setAttrVal("Report_dep", Report_dep);
	}
	/**
	 * 填报医生
	 * @return String
	 */
	public String getReport_doc() {
		return ((String) getAttrVal("Report_doc"));
	}
	/**
	 * 填报医生
	 * @param Report_doc
	 */
	public void setReport_doc(String Report_doc) {
		setAttrVal("Report_doc", Report_doc);
	}
	/**
	 * 患者姓名
	 * @return String
	 */
	public String getName_pat() {
		return ((String) getAttrVal("Name_pat"));
	}
	/**
	 * 患者姓名
	 * @param Name_pat
	 */
	public void setName_pat(String Name_pat) {
		setAttrVal("Name_pat", Name_pat);
	}
	/**
	 * 疾病名称
	 * @return String
	 */
	public String getName_disease() {
		return ((String) getAttrVal("Name_disease"));
	}
	/**
	 * 疾病名称
	 * @param Name_disease
	 */
	public void setName_disease(String Name_disease) {
		setAttrVal("Name_disease", Name_disease);
	}
	/**
	 * 自定义1
	 * @return String
	 */
	public String getDef1() {
		return ((String) getAttrVal("Def1"));
	}
	/**
	 * 自定义1
	 * @param Def1
	 */
	public void setDef1(String Def1) {
		setAttrVal("Def1", Def1);
	}
	/**
	 * 自定义2
	 * @return String
	 */
	public String getDef2() {
		return ((String) getAttrVal("Def2"));
	}
	/**
	 * 自定义2
	 * @param Def2
	 */
	public void setDef2(String Def2) {
		setAttrVal("Def2", Def2);
	}
	/**
	 * 自定义3
	 * @return String
	 */
	public String getDef3() {
		return ((String) getAttrVal("Def3"));
	}
	/**
	 * 自定义3
	 * @param Def3
	 */
	public void setDef3(String Def3) {
		setAttrVal("Def3", Def3);
	}
}